api/logs/views.py

```diff
@@ -171,3 +183,72 @@ def get_object(self):
     # overrides RetrieveUpdateDestroyAPIView
     def perform_destroy(self, instance):
         pass
+
+
+class NodeLogAddedContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin, LogMixin):
+    """List of added contributors that a given log is associated with. *Read-only*.
+
+    Paginated list of users that were added as contributors, associated with a log. Each resource contains the full
+    representation of the user, meaning additional requests to an individual user's detail view are not necessary.
+
+    ##User Attributes
+
+    <!--- Copied Attributes from UserDetail -->
+
+    OSF User entities have the "users" `type`.
+
+        name               type               description
+        ----------------------------------------------------------------------------------------
+        full_name          string             full name of the user; used for display
+        given_name         string             given name of the user; for bibliographic citations
+        middle_names       string             middle name of user; for bibliographic citations
+        family_name        string             family name of user; for bibliographic citations
+        suffix             string             suffix of user's name for bibliographic citations
+        date_registered    iso8601 timestamp  timestamp when the user's account was created
+
+
+    ##Links
+
+    See the [JSON-API spec regarding pagination](http://jsonapi.org/format/1.0/#fetching-pagination).
+
+    ##Actions
+
+    *None*.
+
+    <!--- Copied Query Params from UserList -->
+
+    ##Query Params
+
+    + `page=<Int>` -- page number of results to view, default 1
+
+    + `filter[<fieldname>]=<Str>` -- fields and values to filter the search results on.
+
+    Users may be filtered by their `id`, `full_name`, `given_name`, `middle_names`, or `family_name`.
+
+    + `profile_image_size=<Int>` -- Modifies `/links/profile_image_url` of the user entities so that it points to
+    the user's profile image scaled to the given size in pixels.  If left blank, the size depends on the image provider.
+
+    #This Request/Response
+    """
+
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+
+    required_read_scopes = [CoreScopes.USERS_READ]
+    required_write_scopes = [CoreScopes.NULL]
+
+    serializer_class = UserSerializer
+
+    view_category = 'logs'
+    view_name = 'log-added_contributors'
+
+    # overrides ListAPIView
+    def get_queryset(self):
+        log = self.get_log()
+        added_contrib_ids = log.params.get('contributors')
```

Comment:
Looks like `log` is actually a `website.project.model.NodeLog`, and `log.params` is a `dict`.

```
(Pdb) l
246     
247         # overrides ListAPIView
248         def get_queryset(self):
249             log = self.get_log()
250             import pdb; pdb.set_trace()
251  ->         added_contrib_ids = log.params.get('contributors')
252             if added_contrib_ids is None:
253                 return []
254             added_users = User.find(Q('_id', 'in', added_contrib_ids))
255             return added_users
[EOF]
(Pdb) log
<NodeLog(u'contributor_added', params={u'project': u'ba9zp', u'node': u'498yc', u'contributors': [u'nfqxe']}) with id u'561fbd6d029bdb16f3f32dac'>
(Pdb) log.__class__
<class 'website.project.model.NodeLog'>
(Pdb) log.params
{u'project': u'ba9zp', u'node': u'498yc', u'contributors': [u'nfqxe']}
(Pdb) log.params.__class__
<type 'dict'>
(Pdb)
```